Makefile

```diff
@@ -297,7 +298,7 @@ initiate-release-as: $(STANDARD_VERSION)
 
 .PHONY: kind-cluster
 kind-cluster: $(OPERATOR_SDK)
-	kind create cluster --config hack/kind/config.yaml
+	kind create cluster --config hack/kind/config.yaml --image $(KIND_NODE_IMAGE)
```

Comment:
can't we specify the image in the config itself?

Reply:
@sthaha We can do that as well. But I feel explicit argument is more pragmatic, WDYT?

Reply:
I actually feel the other way, have the image specified in the config.yaml with a comment/explanation to why this is required. The reason being we shouldn't rely on Makefile having this information when the config itself can. 

On related note, I don't see this issue happen in my kind cluster running 1.21.1, did you try increasing the `--timeout`?

Reply:
I tried increasing the `--timeout`, but it didn't help.

Reply:
Updated the config.yaml instead of using a flag(--image).